index.html

```diff
@@ -1,12 +1,161 @@
+            Become the part of this year's concert by buying your ticket online.</p>
+        </div>
+        <div class="home-p2">
+          <span>2012.10.15(FRI) ~ 17(SUN)</span>
+          <p>@Madison Square Garden, New York City, USA</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="home2 container-fluid" id="program">
+      <p>Main Program <img class="line" src="assets/images/line_icon5.png" alt="line"> </p>
+      <div class="card-box container-fluid">
+        <div class="box box1 container-fluid">
+          <div class="icon icon1 no-hover">
+            <img class="no-hover" src="assets/images/music2_icon.svg" alt="microphone">
+          </div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Music</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">Listen to prominent musicians from various countries sharing the message of love through
+              their music.</p>
+          </div>
+        </div>
+        <div class="box box2 container-fluid">
+          <div class="icon icon2 no-hover">
+            <img class="no-hover" src="assets/images/exhibition_icon.svg" alt="exhibition icon">
+          </div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Exhibition</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">There will be a lot of items on exhibition ranging from fashoin to art and promotions.
+            </p>
+          </div>
+        </div>
+        <div class="box box3 container-fluid">
+          <div class="icon icon3 no-hover">
+            <img class="no-hover" src="assets/images/music_icon.svg" alt="music box icon">
+          </div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Entertainment</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">Entertainment is core factor of the Back To Roots Global Concert, and be ready to be
+              blown away.</p>
+
+          </div>
+        </div>
+        <div class="box box4 container-fluid">
+          <div class="icon icon4 no-hover"><img class="no-hover" src="assets/images/donation_icon.svg"
+              alt="dollar coin icon"></div>
+          <div class="head6 no-hover">
+            <h6 class="no-hover">Pledges <br /> & <br /> Donations</h6>
+          </div>
+          <div class="box-p no-hover">
+            <p class="no-hover">Most important part of the Back To Roots Global Concert is donations that will help
+              eliminate some of
+              global challenges through charities.</p>
+          </div>
+        </div>
+      </div>
+      <p class="no-display">SEE THE WHOLE PROGRAM</p>
+      <div class="btn-div card-box"><input type="button" value="Join BTR Global Concert 2012" class="join-btn"> </div>
+    </div>
+  </section>
+
+  <!-- guest artists -->
+  <section class="artists"></section>
+  <section class="partners">
```

Comment:
[REQUIRED] You still didn't include the header for the speakers section.

| Your Design|
|-----------|
|![image](https://user-images.githubusercontent.com/62938735/192120928-bddb5162-d072-4b7d-8a15-132ca510f52a.png)|

[OPTIONAL]  I am only marking this as optional because the previous reviewer missed it. You are expected to include the chessboard design behind each image in the speakers' section


|Template |
|---------|
|![image](https://user-images.githubusercontent.com/62938735/192121121-f341d705-2a05-48d0-b5e3-7739d0c255b5.png)|

[OPTIONAL] There is also supposed to be a dash under the speaker names but you forgot to include them